src/theme/icons/players.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default {
+  path: (
+    <svg
```

Comment:
You don't need an svg tag here. You can move the properties down to the `<path>` tag. Pull the `viewBox` out into it's own top level property like you did on `npcs.js`, and add the `fill="currentColor"` attribute please. :)

You also don't need the `xlmns` property when you move it down to the `<path>` tag. That'll get added by Chakra on the auto-generated `<svg>` tag.